app/workers/repo_clone_worker.rb

```diff
@@ -0,0 +1,21 @@
+require 'open3'
+require 'shellwords'
+
+class RepoCloneWorker
+  include Sidekiq::Worker
+  sidekiq_options queue: :event
+
+  def perform(mission_params)
+    # FIXME: maybe has shell execution vulnerabilities.
+    #        you should validate and escape url of mission params.
+    url = Shellwords.shellescape(mission_params['repository'])
+    dir_name = Shellwords.shellescape(url.match(/\/(.*)*.git/).to_s.gsub('/', ''))
```

Comment:
**[rubocop]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
Performance/StringReplacement: Use `delete` instead of `gsub`.